include/xrpl/protocol/Protocol.h

```diff
@@ -95,6 +95,12 @@ std::size_t constexpr maxDIDAttestationLength = 256;
 /** The maximum length of a domain */
 std::size_t constexpr maxDomainLength = 256;
 
+/** The maximum length of MPTokenMetadata */
+std::size_t constexpr maxMPTokenMetadataLength = 1024;
+
+/** The maximum amount of MPTokenIssuance */
+std::uint64_t constexpr maxMPTokenAmount = 0x7FFFFFFFFFFFFFFFull;
```

Comment:
Stylistic:  I had to put in `'` every four digits to convince myself that there was the proper number of `F`s.

```c++
std::uint64_t constexpr maxMPTokenAmount = 0x7FFF'FFFF'FFFF'FFFFull;
```